nservicebus/msmq/transportconfig.md

```diff
@@ -58,29 +58,25 @@ Often when debugging MSMQ using [native tools](viewing-message-content-in-msmq.m
 
 snippet:ApplyLabelToMessages
 
+## Transactions and delivery guarantees
 
-## Controlling transaction scope options
+MSMQ transport supports all [transaction handling modes](/nservicebus/messaging/transactions.md), i.e. Transaction scope, Receive only, Sends atomic with Receive and No transactions.
 
-The following options can be configured when the MSMQ transport is working in the transaction scope mode.
+Refer to [Transport Transactions](/nservicebus/messaging/transactions.md) for detailed explanation of the supported transaction handling modes and available configuration options. 
 
 
-### Isolation level
+### Transaction scope
 
-NServiceBus will by default use the `ReadCommitted` [isolation level](https://msdn.microsoft.com/en-us/library/system.transactions.isolationlevel).
+In this mode the ambient transaction is started before receiving the message. The transaction encompasses all stages of processing including user data access and saga data access. If all the logical data stores (transport, user data, saga data) use the same physical store there is no escalation to Distributed Transaction Coordinator (DTC).
 
-NOTE: Version 3 and below used the default isolation level of .Net which is `Serializable`.
 
-Change the isolation level using
+### Native transactions
 
-snippet:MsmqTransactionScopeIsolationLevel
+In MSMQ transport there is no distinction between the *ReceiveOnly* and *SendsAtomicWithReceive* levels, they are both handled in an identical way.
 
+The native transaction for receiving messages is shared with sending operations. That means the message receive operation and any send or publish operations are committed atomically.
 
-### Transaction timeout
 
-NServiceBus will use the [default transaction timeout](https://msdn.microsoft.com/en-us/library/system.transactions.transactionmanager.defaulttimeout) of the machine the endpoint is running on.
+### Unreliable (Transactions Disabled)
 
-Change the transaction timeout using
-
-snippet:MsmqTransactionScopeTimeout
-
-Or via .config file using a [example DefaultSettingsSection](https://msdn.microsoft.com/en-us/library/system.transactions.configuration.defaultsettingssection.aspx#Anchor_5).
+In this mode when message is received from an input queue it's immediately removed from it. If processing fails the message is lost, because the operation cannot be rolled back. Also any other operation performed when processing the message is executed outside of the transaction, it can't be rolled back. That might lead to undesired side effects.
```

Comment:
> If processing fails the message is lost, because the operation cannot be rolled back.

I'm notoriously terrible at commas. Is it right to have one here?